ssr/src/component/nav.rs

```diff
@@ -2,7 +2,6 @@ use super::nav_icons::*;
 use leptos::*;
```

Comment:
Lets comment out code we won't use? (TrophyIcon)

Reply:
yes

Reply:
working on separte issue to resolve warning based on best practices